code/modules/atmospherics/machinery/binary/passive_gate.dm

```diff
@@ -59,10 +59,9 @@ obj/machinery/atmospherics/binary/passive_gate
 			if(network2)
 				network2.update = 1
 
-obj/machinery/atmospherics/binary/passive_gate/attackby(obj/weapon as obj, mob/user as mob)
+obj/machinery/atmospherics/binary/passive_gate/attack_hand(mob/user as mob)
 	..()
-	if(istype(weapon, /obj/item/device/multitool))
```

Comment:
Why remove the multitool check?

Reply:
oh I forgot I changed it to test it easier
will add that back in a jiffy